sessions/class-01/exercise/start.js

```diff
@@ -10,7 +10,18 @@
  *
  * @returns {number} Value with "number" type, 0 if not a number.
  */
-export function safelyConvertToNumber(possiblyNum) {}
+export function safelyConvertToNumber(possiblyNum) {
+    if(typeof possiblyNum === 'symbol'){
+        return 0;
+    }
+    let number = Number.parseFloat(possiblyNum);
```

Comment:
```suggestion
    const number = Number.parseFloat(possiblyNum);
```

Since we are not reassigning the variable, so use should use `const`.